diff --git i/cmd/nginx-auth/nginx-auth.go w/cmd/nginx-auth/nginx-auth.go
index 09da74da..9770404a 100644
--- i/cmd/nginx-auth/nginx-auth.go
+++ w/cmd/nginx-auth/nginx-auth.go
@@ -1,7 +1,7 @@
 // Copyright (c) Tailscale Inc & AUTHORS
 // SPDX-License-Identifier: BSD-3-Clause
 
-//go:build linux
+//go:build unix
 
 // Command nginx-auth is a tool that allows users to use Tailscale Whois
 // authentication with NGINX as a reverse proxy. This allows users that
